addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -712,12 +712,21 @@ export function makeActionManager(env, router = _router) {
         delete action.context.no_breadcrumbs;
 
         let resId = viewProps.resId;
+        const topBarActions =
+            view.type === "form" ? [] : context.parentActionChildren || action.children_ids;
+        const fromParentAction = (view.type !== "form" && context.fromParentAction) || false;
+        const parentActionId = topBarActions?.[0]?.parent_action_id[0];
```

Comment:
Can `topBarActions` be undefined?

Reply:
yes, if there is no `parentActionChildren` or `children_ids` in the context, which is happening when no `topbar_actions` is defined